docs/dev/uap/concepts_and_definitions.rst

```diff
@@ -15,7 +15,7 @@ Users
 -----
 
 There are two kinds of users: ``FacilityUser`` and ``DeviceOwner``. A
-``FacilityUser`` is associated with a particular ``Facility``, and the user’s
+``FacilityUser`` is associated with a particular ``Facility``, and the user's
```

Comment:
Realized that I had missed converting GDocs fancy single quotes into normal ones, so fixed those here too.